test/runtests.jl

```diff
@@ -202,6 +202,67 @@ include("setup_tests.jl")
                 @test_throws ArgumentError test_hash(BadHashMethod())
             end
 
+            @testset "Pluto-defined strucst are stable" begin
+                notebook_project_dir = joinpath(@__DIR__, "..")
+                @info "Notebook project: $notebook_project_dir"
+
+                notebook_str = """
+                ### A Pluto.jl notebook ###
+                # v0.19.36
+
+                using Markdown
+                using InteractiveUtils
+
+                # ╔═╡ 3592b099-9c96-4939-94b8-7ef2614b0955
+                import Pkg
+
+                # ╔═╡ 72871656-ae6e-11ee-2b23-251ac2aa38a3
+                begin
+                    Pkg.activate("$notebook_project_dir")
+                    using StableHashTraits
+                end
+
+                # ╔═╡ 1c505fa8-75fa-4ed2-8c3f-43e28135b55d
+                begin
+                    bytes2hex_(x::Number) = x
+                    bytes2hex_(x) = bytes2hex(x)
+                end
+
+                # ╔═╡ b449d8e9-7ede-4171-a5ab-044c338ebae2
+                struct MyStruct end
+
+                # ╔═╡ 1e683f1d-f5f6-4064-970c-1facabcf61cc
+                StableHashTraits.stable_hash(MyStruct()) |> bytes2hex_
+
+                # ╔═╡ Cell order:
+                # ╠═3592b099-9c96-4939-94b8-7ef2614b0955
+                # ╠═72871656-ae6e-11ee-2b23-251ac2aa38a3
+                # ╠═1c505fa8-75fa-4ed2-8c3f-43e28135b55d
+                # ╠═b449d8e9-7ede-4171-a5ab-044c338ebae2
+                # ╠═1e683f1d-f5f6-4064-970c-1facabcf61cc
+                """
+
+                server = Pluto.ServerSession()
+                server.options.evaluation.workspace_use_distributed = false
```

Comment:
Why do you need to set this to false?

Are workspaces named any differently if this is set to true?

Reply:
Nope. This determines whether the workspace operates within the current process or outside of it. Keeping it within makes it easier to debug when running tests line by line.